serialize.go

```diff
@@ -42,6 +47,32 @@ func GetSerializer(key string) (Serializer, bool) {
 	return codec, ok
 }
 
+type IntegerSerializer struct{}
+
+func (IntegerSerializer) EncObject(e *Encoder, v POJO) error {
+	bigInt, ok := v.(bigInteger)
+	if !ok {
+		return e.encObject(v)
+	}
+	bigInt.Signum, bigInt.Mag = bigInt.GetSignAndMag()
+	return e.encObject(bigInt)
+}
+
+func (IntegerSerializer) DecObject(d *Decoder) (interface{}, error) {
+	bigInt, err := d.DecodeValue()
+	if err != nil {
+		return nil, err
+	}
+
+	result, ok := bigInt.(*bigInteger)
+	if !ok {
+		panic("result type is not Integer, please check the whether the conversion is ok")
```

Comment:
@wongoo how about return this to err

Reply:
yes，but i suggest do that in another PR.